src/cranbtree.c

```diff
@@ -147,7 +147,10 @@ cranbtree_t *cbt_clone(cranbtree_t * cbt)
 void *cbt_insert(cranbtree_t * bt, int key, void *object)
 {
 	assert(bt != NULL);
-	assert(object != NULL);
```

Comment:
The error should not be set on an insert. search, delete, update are the only operations that requires the library to search for key (provided by the user) in the tree and this is when the key might not be found.